hypervisor/src/cpu.rs

```diff
@@ -10,6 +10,9 @@
 //
 //
 
+#[cfg(target_arch = "aarch64")]
+use std::sync::Arc;
+
```

Comment:
Nit: please remove the blank line and let clippy format the imports.

Reply:
Fixed.

Reply:
Nightly compiler requires that space:

```
 
 #[cfg(target_arch = "aarch64")]
 use std::sync::Arc;
+
```

Reply:
https://github.com/cloud-hypervisor/cloud-hypervisor/actions/runs/13256271078/job/37003601590?pr=6923